shesmu-pluginapi/src/main/java/ca/on/oicr/gsi/shesmu/plugin/filter/ActionFilterType.java

```diff
@@ -30,4 +30,14 @@ public String[] getTypes() {
   public void setTypes(String[] types) {
     this.types = types;
   }
+
+  @Override
+  public String toString() {
+    StringBuilder writeOut = new StringBuilder();
+    writeOut.append("Action type filter of types: ");
+    for (String type : types) {
+      writeOut.append(type).append(", ");
```

Comment:
Leaves an extra comma at the end, which could be confusing. Could use String.join instead, or delete from the StringBuilder after. Same pattern in a few other places too

Reply:
Fixed